runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/UIActionSlot.java

```diff
@@ -128,8 +129,13 @@ public void mouseExited(MouseEvent mouseEvent)
 		uiLabelActions.setFont(FontManager.getRunescapeSmallFont());
 		uiLabelActions.setForeground(ColorScheme.LIGHT_GRAY_COLOR);
 
+		uiLabelCost = new JShadowedLabel("Unknown");
```

Comment:
I think this should just default to empty/null and the null set block can be removed?